libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -80,6 +80,23 @@ void StaticAnalyzer::endVisit(FunctionDefinition const&)
 	m_localVarUseCount.clear();
 }
 
+bool StaticAnalyzer::visit(ModifierInvocation const& _modifier)
+{
+	if (m_constructor)
+		if (auto decl = dynamic_cast<ContractDefinition const*>(_modifier.name()->annotation().referencedDeclaration))
+			for (auto const& specifier: m_currentContract->baseContracts())
+				if (specifier->name().annotation().referencedDeclaration == decl && !specifier->arguments().empty())
+				{
+					SecondarySourceLocation ssl;
+					ssl.append("Overriden constructor call is here:", specifier->location());
+
+					m_errorReporter.declarationError(
+						_modifier.location(), ssl, "Duplicated super constructor call."
```

Comment:
I think our coding convention is to put those arguments one line each.

Reply:
Yes, missed it. Fixed now.